### PR TITLE
Addressing Issue #116 and one item in issue #119

### PR DESCRIPTION
**Pull request by jhdalek55**
I've now gone through and have made sure  instances of proper names for roles, ECUs, and metadata are capitalized. I also fixed some inconsistent capitalization in headings and subheads. Now only the first word in headings is capitalized, except where proper nouns appear. This addresses  Issue #116.

I also addressed the one element in Issue #119. That is I've included the missing definition of "metadata." For this, I followed Ira's suggestion and included both the NIST definition and the version proposed by @mcvand in discussion on the issue.
I also added definitions for three acronyms used in the document that were never defined.
Lastly I fixed a handful of sentences that to my eye had structural.
I was going to add the citations to our definitions at the beginning of the document, per @iramcdonald. But I don't know how to set up the citations in Markdown using this particular format. If someone can let me know how, I can take care of that, but probably not until early next week. If another team member could step in and do this, I would likely be faster.

### CODE REVIEW

File: uptane-standard.md [hunk 1]
```diff
@@ -238,23 +248,23 @@ A standards document that can guide the safe design, integration and deployment
 * This added design flexibility, however, could be a liability if the framework is implemented incorrectly.
 * Standardization of crucial steps in the design, implementation and use of Uptane can assure that customizability does not impact security or functionality.
 
-## Scope of Standards Coverage
+## Scope of Standards coverage
```
Comment (mnm678) [inline uptane-standard.md]: Should 'standards' also be lowercase here?

File: uptane-standard.md [hunk 2]
```diff
@@ -460,45 +470,45 @@ A repository's Root metadata distributes the public keys of the top-level Root,
...
+In addition to the required metadata, Targets metadata files MAY contain extra metadata for images on the repository. This metadata can be customized for a particular use case. Examples of use cases for different types of custom metadata can be found in the Deployment Considerations document. However, there are a few important pieces of custom metadata that SHOULD be present in most implementations. In addition, there is one element in the custom metadata that MUST be present in the Targets metadata from the Director.
 
-Custom metadata also MAY contain a field or section that is demarcated as custom metadata that MUST match whenever two pieces of metadata are checked against each other--most commonly, when targets metadata from the Director and Image repositories are checked against each other.
+Custom metadata also MAY contain a field or section that is demarcated as custom metadata, and this field MUST match whenever two pieces of metadata are checked against each other. Most commonly, this occurs when targets metadata from the Director and Image repositories are checked against each other.
```
Comment (mnm678) [inline uptane-standard.md]: Not all custom metadata must match. The must match field/section would be a subsection of the custom metadata (although I agree this was a bit confusing as written before).

File: uptane-standard.md (hunk 1 above)
Comment (jhdalek55) [reply uptane-standard.md]: I think nit, because its referring to this particular standard document.

File: uptane-standard.md (hunk 2 above)
Comment (jhdalek55) [reply uptane-standard.md]: I went back and forth as to how to fix this. I avoided edits for the most part in this pass except where the syntax was confusing. This sentence WAS confusing, because it is saying that within custom metadata there is a field called custom metadata. I thought my rewrite, which noted that only "this field" --meaning the added field or section, was what had to match, made the distinction you mentioned. If you think you can rephrase the idea in a clearer manner, please do so.

**Comment by jhdalek55** [comment]
One other question---In section 5.4.2 What the primary does---The text says "...it SHALL perform the following seven steps." But, step number 5--Send the latest time--is marked as "optional." I don't think we can include a step marked "optional" in a section where the word "SHALL" is used.

I would suggest we change the test to "shall perform the following six steps" and bullet #5 be  made a stand alone sentence placed after the bulleted list saying, "If the secondary has no access to the current time, the Primary should send the latest time to the Secondaries, before sending the metadata or images."

File: uptane-standard.md (hunk 2 above)
Comment (tkfu) [reply uptane-standard.md]: Yeah, neither of these changes are ok.

* The first sentence needs to be reverted; you've turned a dependent clause ([...] custom metadata **_that_** MUST match [...]) into an independent clause ([...] custom metadata, **_and_** this field MUST match), and that changes the meaning of the sentence in a way that is somewhere between unclear and incorrect.
* The referent of "this occurs" in the second sentence can't be gleaned easily from context--is "this" the existence of the demarcated field, or the checking of metadata from two sources against each other? Normally, without further clarification (or grammatical disambiguation), we expect a referent to refer to the subject of the previous sentence, which would lead to the wrong conclusion in this case. If you think about it thoroughly, you can figure out what the referent is supposed to be, but you can only get there if you already understand what's going on.

File: uptane-standard.md [hunk 3]
```diff
@@ -460,45 +470,45 @@ A repository's Root metadata distributes the public keys of the top-level Root,
...
 
-The following information SHOULD be provided for each image on both the Image repository and the Director repository, and SHOULD be included in the "MUST match" section, if it is implemented:
+The following information SHOULD be provided for each image on both the Image repository and the Director repository and, if implemented, SHOULD be included in the "MUST match" section:
```
Comment (tkfu) [inline uptane-standard.md]: I agree with the spirit of cleaning this up, but I think we ought to explicitly clarify that there are two different statements here: (1) this information SHOULD be provided, and (2) if the **_MUST match_** section is implemented, that is the correct place for this information. They way it reads (in both the previous version and in the proposed change), it sounds like we're saying "if you choose to provide this information, it SHOULD go in a MUST match section". That conflicts with the previous paragraph (because having a MUST match section at all is a MAY, not a SHOULD).

File: uptane-standard.md [hunk 4]
```diff
@@ -561,12 +571,12 @@ There is a difference between the file name in a metadata file or an ECU, and th
...
 
 For example:
 
-* The version number of the Snapshot metadata file is 61, and its filename in the Timestamp metadata is "snapshot.json". The filename on the repository will be "61.snapshot.json".
-* There is an image with the filename "acme_firmware.bin" specified in the Targets metadata, with a SHA256 of "aaaa" and a SHA512-256 of "bbbb". It will have two filenames on the repository: "aaaa.acme_firmware.bin" and "bbbb.acme_firmware.bin".
+* The version number of the Snapshot metadata file is 61, and its filename in the Timestamp metadata is "snapshot.json." The filename on the repository will be "61.snapshot.json."
+* There is an image with the filename "acme_firmware.bin" specified in the Targets metadata, with a SHA256 of "aaaa" and a SHA512-256 of "bbbb." It will have two filenames on the repository: "aaaa.acme_firmware.bin" and "bbbb.acme_firmware.bin."
```
Comment (tkfu) [inline uptane-standard.md]: Needs to be reverted (the changes in both the previous lines). The quotation marks here are not for quoting text; they are for demarcating the specific syntax of a filename. Moving the period inside the quotation marks is incorrect.

File: uptane-standard.md [hunk 5]
```diff
@@ -627,17 +637,17 @@ The inventory database MUST record the following pieces of information:
     * The vehicle identifier the ECU is associated with
     * An ECU key (symmetric or asymmetric; for asymmetric keys, only the public part SHOULD be stored)
     * The ECU key identifier (as defined in {{common_metadata}})
-    * Whether the ECU is a primary or a secondary
+    * A notation of whether the ECU is a Primary or a Secondary
```
Comment (tkfu) [inline uptane-standard.md]: This isn't just a verbiage change; it changes the requirements.

We had this discussion several months ago: the information that the inventory database needs to have is whether the ECU is a primary or a secondary. That could be indicated by a notation, yes, like a boolean flag. But it could also be indicated by something that is not notational, like which table the ECU is in.

File: uptane-standard.md [hunk 6]
```diff
@@ -822,11 +832,11 @@ In order to perform partial verification, an ECU SHALL perform the following ste
...
 
-Before starting full verification, the repository mapping metadata MUST be consulted to determine where to download metadata from. This procedure assumes the basic Uptane case: there are only two repositories (Director and Image), and all image paths are required to be signed by both repositories. If a more complex repository layout is being used, refer to {{DEPLOY}} for guidance on how to determine where to download metadata from.
+Before starting full verification, it must be determined from where metadata should be downloaded. For this task, the repository mapping metadata MUST be consulted. This procedure assumes the basic Uptane case: there are only two repositories (Director and Image), and all image paths are required to be signed by both repositories. If a more complex repository layout is being used, refer to {{DEPLOY}} for guidance on how to determine where metadata should be downloaded from.
```
Comment (tkfu) [inline uptane-standard.md]: The changes in the first sentence make it read worse to my eyes, in particular the way it's changed to the passive voice. Is there any reason for the change apart from an aversion to ending sentences with prepositions?

File: uptane-standard.md [hunk 7]
```diff
@@ -839,26 +849,26 @@ In order to perform full verification, an ECU SHALL perform the following steps:
...
 1. Download and check the top-level Targets metadata file from the Image repository, following the procedure in {{check_targets}}.
-1. Verify that Targets metadata from the Director and Image repositories match. A primary ECU MUST perform this check on metadata for all images listed in the Targets metadata file from the Director repository downloaded in step 6. A secondary ECU MAY elect to perform this check only on the metadata for the image it will install. (That is, the target metadata from the Director that contains the ECU identifier of the current ECU.) To check that the metadata for an image matches, complete the following procedure:
+1. Verify that Targets metadata from the Director and Image repositories match. A Primary ECU MUST perform this check on metadata for all images listed in the Targets metadata file from the Director repository downloaded in step 6. A Secondary ECU MAY elect to perform this check only on the metadata for the image it will install. (That is, the Targets metadata from the Director that contains the ECU identifier of the current ECU.) To check that the metadata for an image matches, complete the following procedure:
```
Comment (tkfu) [inline uptane-standard.md]: In the parenthetical here, it actually is "target metadata" we're talking about, not "Targets metadata". That is, it's the metadata about the specific target within the Targets metadata. That's definitely not very clear, though. We could change it to something like `(That is, the metadata for the particular target within the Targets metadata from the Director that contains the ECU identifier of the current ECU.)` Might be a bit wordy, though.

File: uptane-standard.md (hunk 3 above)
Comment (jhdalek55) [reply uptane-standard.md]: Ok. Got it. So, let's  use what you have noted above: 

The information listed below SHOULD be provided for each image on both the Image repository and the Director repository. If  a "MUST match section" is to be implemented, that is where this information should be placed."

Comment (tkfu) [reply uptane-standard.md]: Sounds good, except that the SHOULD should be capitalized.

File: uptane-standard.md (hunk 2 above)
Comment (jhdalek55) [reply uptane-standard.md]: Ok. Initially I had two problems with these two sentences. The first is the use of the phrase "custom metadata" twice in the sentence, seemingly referring to a large section and a subsection  simultaneously. Secondly, the addition separating the two phrases with a dash struck me here as sloppy and less than clear. But, I understand your point about accidentally changing the meaning by changing the nature of the clause. So, let me try this another way. What if I take out the repeated reference to "custom metadata"?

Custom metadata MAY also contain a demarcated field or section that MUST match whenever two pieces of metadata are checked against each other, such as when Targets metadata from the Director repository is checked against metadata from the Image repository.

File: uptane-standard.md (hunk 3 above)
Comment (jhdalek55) [reply uptane-standard.md]: The second SHOULD? Ok, I can fix that.

File: uptane-standard.md (hunk 4 above)
Comment (jhdalek55) [reply uptane-standard.md]: Ok. Sorry about that. I will fix those.

File: uptane-standard.md (hunk 5 above)
Comment (jhdalek55) [reply uptane-standard.md]: Ok. I can put it back. What bothered me was that everything else in the list is a noun (a thing). This one was not. It was a parallelism problem. But if there's no getting around it, I'm change it back. Sorry if its a repeat error. This was the first pass all through the document I've made in months , so I hadn't recalled that discussion.

File: uptane-standard.md (hunk 6 above)
Comment (jhdalek55) [reply uptane-standard.md]: Mostly the latter. Though its no longer an iron clad rule in grammar, sentences ending with prepositions read rather awkwardly---though I admit, the change doesn't improve it much. I'll put that back too. Particularly at this stage I'm not looking to change things just to hange them.

File: uptane-standard.md (hunk 7 above)
Comment (jhdalek55) [reply uptane-standard.md]: I hesitated on changing this because I thought it might be referring a target in the generic sense. Let's keep it simple. I'll put it back to the original.

File: uptane-standard.md (hunk 2 above)
Comment (jhdalek55) [reply uptane-standard.md]: Fixed. Let me know if this addresses your concerns.

File: uptane-standard.md (hunk 3 above)
Comment (jhdalek55) [reply uptane-standard.md]: FIXED

File: uptane-standard.md (hunk 4 above)
Comment (jhdalek55) [reply uptane-standard.md]: FIXED.

File: uptane-standard.md (hunk 5 above)
Comment (jhdalek55) [reply uptane-standard.md]: FIXED.

File: uptane-standard.md (hunk 6 above)
Comment (jhdalek55) [reply uptane-standard.md]: FIXED.

File: uptane-standard.md (hunk 7 above)
Comment (jhdalek55) [reply uptane-standard.md]: FIXED.

**Comment by jhdalek55** [comment]
We have a conflict now, probably because Trishank merged Max's PR while I was making my corrections.

File: uptane-standard.md (hunk 1 above)
Comment (jhdalek55) [reply uptane-standard.md]: Standards here is a proper noun, which I do capitalize throughout. Would it look less weird if I changed it to Uptane Standards coverage?

**Comment by jhdalek55** [comment]
Ok. I fixed the merge conflict and made all changes. @tkfu and @mnm678. If all is now OK, can we merge?
